### PR TITLE
DOC-5718 - fixed instances of wrong versions in docs.

### PR DESCRIPTION
**Pull request by paulmoloneyr3**
This includes fixing wrong versions in text and in links. Also fixed links where it was pointing to an old version of a topic in the archived docs, where that topic actually exists in the same version of the current docs.

### CODE REVIEW

**Review by yarivus — APPROVED** [review]
LGTM

**Review by yarivus — APPROVED** [review]
LGTM

File: content/en/platform/corda/4.10/enterprise/node/operating/shell.md [hunk 1]
```diff
@@ -262,7 +262,7 @@ You can use the shell to:
 The shell interacts with the node by issuing RPCs (remote procedure calls). You make an RPC from the shell by typing `run`, followed by the name of the desired RPC method.
 
 You can find a list of the available RPC methods
-[here](https://docs.corda.net/api/kotlin/corda/net.corda.core.messaging/-corda-r-p-c-ops/index.html).
+[here](../../../../../en/api-ref/corda/4.10/community/kotlin/corda/net.corda.core.messaging/-corda-r-p-c-ops/index.html).
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.10/enterprise/node/operating/shell.md:265]: Is this right? Shouldn't need the `en`?

Comment (paulmoloneyr3) [reply content/en/platform/corda/4.10/enterprise/node/operating/shell.md:265]: Slightly superfluous!

**Review by nadinequinr3 — APPROVED** [review]

**Pull request merged**